src/seedu/addressbook/storage/StorageFile.java

```diff
@@ -88,6 +88,12 @@ public void save(AddressBook addressBook) throws StorageOperationException {
         /* Note: Note the 'try with resource' statement below.
          * More info: https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html
          */
+    	
+    	if(!path.toFile().exists()){
+    		//If storage file is not found or deleted, print out warning.
+    		System.out.println("Why you no have storage file?");
```

Comment:
- I'm glad you found the correct place to check for the exception.

- But a system.out with the warning text is NOT exception handling.